src/lib/wallet.js

```diff
@@ -186,9 +199,6 @@ export default {
             }
             return address;
           },
-          sign: (data) => store.dispatch('accounts/sign', data),
-          signTransaction: (txBase64, opt) =>
-            store.dispatch('accounts/signTransaction', { txBase64, opt }),
```

Comment:
This removes the confirmation of generation the invite link, is it an intended change?